index.html

```diff
@@ -758,25 +758,33 @@ <h3>Credentials</h3>
 <a>verifiable credentials</a>. <a href="#info-graph-vc"></a> below shows a
 more complete depiction of a <a>verifiable credential</a>, which is normally
 composed of at least two information <a>graphs</a>. The first <a>graph</a>
+(the <a>verifiable credential graph</a>, in this case the <a>default graph</a>)
 expresses the <a>verifiable credential</a> itself, which contains credential
-metadata and <a>claims</a>. The second <a>graph</a> expresses the digital proof,
-which is usually a digital signature.
+metadata and other <a>claims</a>. The second <a>graph</a> 
+(the <a>proof graph</a> of the <a>verifiable credential</a>, which is 
+a <a>named graph</a>) expresses the digital proof, which is, in this case, a digital signature.
         </p>
 
         <figure id="info-graph-vc">
           <img style="margin: auto; display: block; width: 100%;"
-               src="diagrams/vc-graph.svg" alt="diagram with a
-               Credential Graph on top connected via a proof to a
-               Proof Graph on the bottom.  The Credential Graph has
-               Credential 123 with 4 properties: 'type' of value
+               src="diagrams/vc-graph.svg" alt="Diagram with a
+               collections of claims for a 'verifiable credential graph'
+               on top connected via a proof to a
+               'verifiable credential proof graph' on the bottom.  
+               The claims for a verifiable credential include
+               'Credential 123' as a subject with 4 properties: 'type' of value
                ExampleAlumniCredential, 'issuer' of Example University,
                'validFrom' of 2010-01-01T19:23:24Z, and
-               credentialSubject of Pat, who has an alumniOf property
-               with value of Example University.  The Proof Graph has
-               Signature 456 with 5 properties: 'type' of
+               credentialSubject of Pat, who also has an alumniOf property
+               with value of Example University.  The verifiable credential proof graph 
+               has and object 'Signature 456' subject with 5 properties: 'type' of
```

Comment:
I know `and` should be `an`, but I'm not sure I understand this sentence, so don't want to try to reword it immediately. I *think* it should say something like, "The verifiable credential proof graph has a `proof` predicate with 'Signature 456' as its  object; the subject 'Signature 456' itself has 5 properties: ..." but I don't think that this text nor the associated diagram is optimal yet.

I think each named graph (including what is currently labeled as "the default graph") should be identified, explicitly, via an example URI or a blank node identifier.
```suggestion
               has an object 'Signature 456' subject with 5 properties: 'type' of
```